src/app/learn/get-started/advanced/route-params/index.html

```diff
@@ -40,7 +40,7 @@
     </li>
     <li>
       <p>
-        In <stache-code>about.service.ts</stache-code>, add data to the shared service so that multiple components can use it. First, import <stache-code>Injectable</stache-code> from Angular. Then copy the <stache-code>AboutService</stache-code> data from the <stache-code>about.component.ts</stache-code> file in <stache-code>src/app/about</stache-code> and update it to replace the public <stache-code>team</stache-code> method with a private <stache-code>users</stache-code> method and add <stache-code>id</stache-code> properties for each user. Then expose public methods to read the data.
+        In <stache-code>about.service.ts</stache-code>, add data to the shared service so that multiple components can use it. First, import <stache-code>Injectable</stache-code> from Angular. Then copy the <stache-code>AboutService</stache-code> data from the <stache-code>about.component.ts</stache-code> file in <stache-code>src/app/about</stache-code>. Update the data to replace <stache-code>team</stache-code> with <stache-code>users</stache-code>, and add <stache-code>id</stache-code> properties for each user. Then expose the public read-only properties.
```

Comment:
Do you mind showing the feedback given?  Technically, the public properties are not read-only.  I'm interested in the problem this addresses.